lib/src/client.dart

```diff
@@ -50,6 +50,10 @@ abstract class Client {
   ///
   void setToken(String token);
 
+  /// Set allows updating connection data.
+  ///
+  void setData(List<int> data);
```

Comment:
Do you need both `getData` callback and `setData` setter? Can we go only with callback only (in this case we will match interface in centrifuge-js - https://github.com/centrifugal/centrifuge-js/blob/9749053dfb669cd559cf0dd50b3f3b7f209a0e02/src/types.ts#L103).

Reply:
@FZambia, basically, only a callback is needed, a getter and setter are not needed, I will do it like in the js version.

Reply:
@FZambia, i made corrections, please look again

Reply:
Thanks 👍